inc/db_pgsql.php

```diff
@@ -676,13 +681,13 @@ function field_exists($field, $table)
 	/**
 	 * Add a shutdown query.
 	 *
-	 * @param resource The query data.
-	 * @param string An optional name for the query.
+	 * @param resource $query The query data.
+	 * @param string $name An optional name for the query.
 	 */
-	function shutdown_query($query, $name=0)
+	function shutdown_query($query, $name="")
 	{
 		global $shutdown_queries;
-		if($name)
+		if(!empty($name))
```

Comment:
`!empty()` negation also checks `isset()` first which is not needed here. So it's a little overkill. Doesn't almost matter here, I know, but it's not really worth changing.